snuba/subscriptions/executor_consumer.py

```diff
@@ -373,6 +388,22 @@ def __init__(
         self.__max_buffer_size = 10000
         self.__closed = False
 
+    def __throttled_commit(self, force: bool = False) -> None:
+        # Commits all offsets and resets self.__commit_data at most
+        # every COMMIT_FREQUENCY_SEC. If force=True is passed, the
+        # commit frequency is ignored and we immediately commit.
+
+        now = time.time()
+
+        if (
+            self.__last_committed is None
+            or self.__last_committed + COMMIT_FREQUENCY_SEC < now
```

Comment:
This feels more readable.
```suggestion
            or now - self.__last_committed >= COMMIT_FREQUENCY_SEC
```